stop_and_wait_plugin/src/stop_and_wait_plugin.cpp

```diff
@@ -115,6 +115,10 @@ namespace stop_and_wait_plugin
             }
         }
 
+        if(current_downtrack < maneuver_plan[0].stop_and_wait_maneuver.start_dist){
```

Comment:
Hmm, this works as a bandaid but means that our plan shrinks pretty badly as we approach the junction between the maneuver before stop&wait and the stop&wait maneuver itself, right? Ideally we'd want them concatenated with the stop&wait maneuver starting at the end of whatever the prior maneuver is.

Reply:
@adev4a Any thoughts on that ^